katana/config.json

```diff
@@ -1,10 +1,10 @@
 {
     "engineer": "",
-    "pythonsrcdir": "",
-    "xmldir": "",
-    "testsuitedir": "",
-    "projdir": "",
-    "idfdir": "",
-    "testdata": "",
+    "pythonsrcdir": "/data/users/apathapa/fnc/python3/2129/warriorframework_py3/warrior",
```

Comment:
why the path is hard-code ?

Reply:
I have made necessary changes and committed